.github/workflows/set-automatic-deploys-enabled.yaml

```
@@ -0,0 +1,37 @@
+name: Set automatic_deploys_enabled (optionally image_tag too)
```

Optional improvement - we could this a nice human readable name:
```suggestion
name: Set automatic deploys
```